packages/components/src/Combobox/components/ComboboxOption/ComboboxOption.tsx

```diff
@@ -21,10 +21,11 @@ export function ComboboxOption(props: ComboboxOptionProps) {
       data-selected={isSelected}
       role="option"
       aria-selected={isSelected}
-      onClick={() =>
+      onClick={() => {
         selectionHandler &&
-        selectionHandler({ id: props.id, label: props.label })
-      }
+          selectionHandler({ id: props.id, label: props.label });
```

Comment:
I like the more elegant `?.()` syntax

wanna swap the `selectionHandler?.(` to that too?